README.md

```diff
@@ -1,11 +1,11 @@
 # BU Slideshow #
-**Contributors:** inderpreet99, awbauer, clrux, mgburns, gannondigital  
-**Tags:** slideshow, images, boston university, bu  
-**Requires at least:** 3.5  
-**Tested up to:** 4.6.1  
-**Stable tag:** 2.3.8  
-**License:** GPLv2 or later  
-**License URI:** http://www.gnu.org/licenses/gpl-2.0.html  
+**Contributors:** inderpreet99, awbauer, clrux, mgburns, gannondigital
```

Comment:
the trailing whitespace here and down to "License URI" guarantee that each line rendered by github on a new line. without trailing whitespaces this page looks like this:
![image](https://user-images.githubusercontent.com/23388231/42712625-518058e6-86ba-11e8-9bc5-e99dc0d1cc20.png)